prost-build/src/code_generator.rs

```diff
@@ -308,18 +308,26 @@ impl<'a> CodeGenerator<'a> {
         let deprecated = self.deprecated(&field);
         let optional = self.optional(&field);
         let ty = self.resolve_type(&field, fq_message_name);
+        let field_wrapper = self
+            .config
+            .field_wrappers
+            .get_first_field(fq_message_name, field.name())
+            .copied();
```

Comment:
In my opinion it is interesting to add a filter here.
Since you will never want an enum wrapped in arc, or an integer
Since they are much cheaper to clone than arc itself, in addition to implementing automatic copying